lib/data/benefit_cap_data.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I would've preferred to see this commit combined with the next one ("Implement getting benefit cap data from YAML in the flow") so that it's clear to see where these weekly caps come from.